src/entities/protocols/seaport.ts

```diff
@@ -122,6 +132,15 @@ export class SeaportTrade extends NFTTrade<SeaportData> {
     return totalPrice
   }
 
+  private commandMap(version: SeaportVersion): CommandType {
+    switch (version) {
+      case SeaportVersion.ONE_POINT_ONE:
```

Comment:
nit (can merge without!): might be less annoying for integrators to write out `SeaportVersion.V1_1` or `SeaportVersion.V1_4`